app/src/main/java/com/example/feelgoodinc/services/UserService.java

```diff
@@ -0,0 +1,135 @@
+package com.example.feelgoodinc.services;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.IBinder;
+import com.example.feelgoodinc.models.User;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+
+/***
+ * Accesses the firebase database to receive and add user data
+ * Used for log in, sign up and change password
+ */
+public class UserService extends Service {
```

Comment:
It'd be good to have an example of how to use the service here in the Javadoc. Surround the code example with `<pre>  </pre>`